opensuse-leap-42.3-ppc64le-openstack.json

```diff
@@ -0,0 +1,51 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "c<wait5>",
+        "linux /boot/ppc64le/linux netsetup=dhcp install=cd:/<wait>",
+        " lang=en_US autoyast=http://{{ .HTTPIP }}:{{ .HTTPPort }}/opensuse-leap-42.3/autoinst.xml<wait>",
+        " textmode=1<enter>",
+        "initrd /boot/ppc64le/initrd<enter>",
+        "boot<enter><wait>"
+      ],
+      "boot_wait": "6s",
+      "disk_size": "3072",
+      "headless": true,
+      "vnc_bind_address":"0.0.0.0",
+      "http_directory": "http",
+      "iso_checksum": "6e6b6e72a8a49020fba25da3a36e72d898952d56f1913d752d073c9b2ff75bab",
+      "iso_checksum_type": "sha256",
+      "iso_url": "http://download.opensuse.org/ports/ppc/distribution/leap/42.3/iso/openSUSE-Leap-42.3-DVD-ppc64le-Build0130-Media.iso",
```

Comment:
Any reason not to use the net installer? downloading a 4.7 GB iso doesn't seem ideal

Reply:
I don't recall why we can't do it this way but I can give it a try and see how it goes.